services/githubService.js

```diff
@@ -25,6 +75,24 @@ const fetchPRsByUser = async (username) => {
   }
 }
 
+const fetchOpenPRs = async () => {
```

Comment:
JSDoc annotations missing

Reply:
Added in the review changes. Thank you for catching this.